cmd/monitoring/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/gagliardetto/solana-go/rpc"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
+)
+
+func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	cfg, err := monitoring.ParseConfig()
+	if err != nil {
+		log.Fatalf("failed to parse configuration: %v", err)
+	}
+
+	http.Handle("/metrics", promhttp.Handler())
+	go func() {
+		if err := http.ListenAndServe(cfg.Http.Address, nil); err != nil {
+			log.Fatalf("failed to start http server with address %s: error %v", cfg.Http.Address, err)
+		}
+	}()
+
+	client := rpc.New(cfg.Solana.RPCEndpoint)
+
+	schemaRegistry := monitoring.NewSchemaRegistry(cfg.SchemaRegistry)
+	trSchema, err := schemaRegistry.EnsureSchema("transmission", monitoring.TransmissionAvroSchema)
```

Comment:
subject should be topic name + value https://github.com/smartcontractkit/gokit/blob/ca2df4d6266ca7a27cb8ba92a29a247d150778c9/kafka/schema_registry.go#L29

Reply:
Got it. Will update!